scripts/tasks/version-check.js

```diff
@@ -32,6 +74,16 @@ Object.keys(versions).forEach(function(name) {
   }
 });
 
+if (!gte(localYarnVersion, MIN_YARN_VERSION)) {
+  allVersionsMatch = false;
+  console.log(
```

Comment:
Shouldn't we also fail the install in this case?

Reply:
Yes, if local yarn version is lte the `MIN_YARN_VERSION`, the process will exit. I don't know whether I got what you mean.

Reply:
Oh I see. I think changing `allVersionsMatch` is confusing since it's not what the original test was about. In fact let's break this task apart into two files? One would be `check-react-versions.js` and another would be `check-development-environment.js`.

Reply:
first check development environment and then check react version?

Reply:
Yep.